lib/Interpreter/Value.cpp

```diff
@@ -231,9 +231,15 @@ namespace cling {
   namespace valuePrinterInternal {
     std::string printTypeInternal(const Value& V);
     std::string printValueInternal(const Value& V);
+    extern const char* const kUndefined;
   } // end namespace valuePrinterInternal
 
   void Value::print(llvm::raw_ostream& Out, bool Escape) const {
+    if (!m_Interpreter) {
+      Out << valuePrinterInternal::kUndefined << "\n";
```

Comment:
I want an empty `Value` to have a unique string representation that is different from error cases. I do not care whether you rename that to e.g. `"<empty>"` or leave it at `"<<<invalid>>>"`. But the output must be different from the output due to an infrastructure issue.